SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md

```diff
@@ -1,48 +1,49 @@
+    * [Get Started: 3 - Set up the SpatialOS GDK Starter Template]({{urlRoot}}/content/get-started/gdk-template)
 
-* Make sure your Spatial CLI is up to date: from a terminal window, run the command `spatial update`. 
-<!-- // TODO: Update the set up link when ready -->
+* Open a terminal window and run the command `spatial update` to ensure your [spatial CLI]({{urlRoot}}/content/glossary#spatial-command-line-tool-cli) installation is up to date. 
 
 ### Terms used in this guide
-`<GameRoot>` - The folder containing your project's `.uproject` file and `Source` folder.  
-`<ProjectRoot>` - The folder containing your `<GameRoot>`.  
-`<YourProject>` - Name of your game project's `.uproject` (for example, `\<GameRoot>\TP_SpatialGDK.uproject`).
+`<GameRoot>` - The directory containing your project's `.uproject` file and `Source` directory.  
+`<ProjectRoot>` - The directory containing your `<GameRoot>`.  
+`<YourProject>` - The name of your project's `.uproject` file (for example, `\<GameRoot>\TP_SpatialGDK.uproject`).
 
 ## Port your game to the GDK
 
 ### 1. Set up the project structure
-1. Ensure you have a `<ProjectRoot>`. If your `<GameRoot>` lives inside of a self-contained folder already, this is your `<ProjectRoot>`. If not, you should create a new folder to represent your `<ProjectRoot>` and move your `<GameRoot>` inside of it.  
+
+
+1. Create a new empty directory to represent your `<ProjectRoot>` and move your `<GameRoot>` directory inside of it.  
    
-    Your project structure should take the form of `\<ProjectRoot>\<GameRoot>\<YourProject>.uproject`
+    Your project structure should be:  `\<ProjectRoot>\<GameRoot>\<YourProject>.uproject`<br/>
+    
     For example:
     `\MyProject\Game\TP_SpatialGDK.uproject`
     
-    > This step is essential as the `spatial` folder must be located in the directory above your `<GameRoot>`. This is so that the GDK scripts work correctly with Unreal.
-1. Your game's project needs some extra files and folders to run with the GDK; you can copy these from the template project that you set up earlier in the [Before you start](#before-you-start) section.
+1. Your project needs some extra files and folders to run with the GDK. Copy these files from the template project that you set up earlier in the [Before you start](#before-you-start) section.
 
-    To do this: either in a terminal window or your file manager, navigate to the root of the `StarterTemplate` and copy all of the files and folders below to your `<ProjectRoot>`:  
+    To do this: either in a terminal window or your file manager, navigate to the root of the `StarterTemplate` repository and copy all of the files and directories below to your `<ProjectRoot>`:  
 
     ``` cpp
-    \TP_SpatialGDK\spatial-resources\spatial\
-    \TP_SpatialGDK\spatial-resources\LaunchClient.bat 
-    \TP_SpatialGDK\spatial-resources\LaunchServer.bat
-    \TP_SpatialGDK\spatial-resources\ProjectPaths.bat
+    \TP_SpatialGDK\spatial\
```

Comment:
These need the `spatial-resources` folder added

Reply:
There's also two new scripts in this directory - `DeployGame.bat` and `BuildProject.bat`. Is there a better way to detail them all?

Reply:
Does `spatial-resources` need to be added or removed? Bit confused. Just had a look at my template project and there isn't a `spatial-resources` directory, must've overlooked this. 
Unfortunately, I don't think there's a better way to list all of the files you need to copy over. 

I'll get the new .bat scripts added to the list.

Reply:
Yes, these need the `spatial-resources` folder added back in.